### PR TITLE
Fixed find target button bug

### PR DESCRIPTION
**Pull request by bisht-richa**
## What does this PR change?
- Fix the `Find target` button behavior for the feature salt > remote commands
- Fix the missing background color for the pending status badge and show/hide the response labels. 

**add description**

## GUI diff

No difference.

Before:
Screenshot 2024-08-20 at 13 48 26

After:
Screenshot 2024-08-20 at 13 47 17

- [ ] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite

- No tests: already covered

- [x] **DONE**

## Links

Fixes: https://github.com/SUSE/spacewalk/issues/25108

Port(s): # **add downstream PR(s), if any**

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: web/html/src/branding/css/base/theme.scss [hunk 1]
```diff
@@ -1332,6 +1332,8 @@ tr.changed > td {
 
 .badge {
   font-size: 12px;
+  border-radius: 10px;
+  background: $eos-bc-gray-1000;
```
Comment (Etheryte) [inline web/html/src/branding/css/base/theme.scss:1336]: I'm guessing this should probably be `$badge-bg`?

Comment (Etheryte) [inline web/html/src/branding/css/base/theme.scss:1336]: We probably need the same fix in the old theme files as well, no?

Comment (bisht-richa) [reply web/html/src/branding/css/base/theme.scss:1336]: It is working fine in the old theme.

Comment (bisht-richa) [reply web/html/src/branding/css/base/theme.scss:1336]: Fixed

**Review by Etheryte — APPROVED** [review]
Looks good, thanks for fixing this. 👍

**Pull request merged**